pkg/utils/selector.go

```diff
@@ -131,17 +131,24 @@ func SelectPods(ctx context.Context, c client.Client, selector v1alpha1.Selector
 // TODO: support to check fieldsSelector
 func CheckPodMeetSelector(pod v1.Pod, selector v1alpha1.SelectorSpec) (bool, error) {
 	if len(selector.Pods) > 0 {
+		meet := false
+	Loop:
 		for ns, names := range selector.Pods {
 			if pod.Namespace != ns {
 				continue
 			}
 
 			for _, name := range names {
 				if pod.Name == name {
-					return true, nil
+					meet = true
+					break Loop
 				}
 			}
 		}
+
+		if !meet {
```

Comment:
Don't need `Loop` label. when leaving the inner for-block, if meet is false boolean, it has no chance to become true.